src/Wallabag/CoreBundle/Resources/views/themes/baggy/Config/index.html.twig

```diff
@@ -100,6 +100,14 @@
             </div>
         </fieldset>
 
+        <fieldset class="w500p inline">
+            <div class="row">
```

Comment:
And maybe we can display or even better disable this widget in case of the two factor isn't enabled